src/routes/safe/components/Transactions/TxList/index.tsx

```diff
@@ -33,14 +42,20 @@ const GatewayTransactions = (): ReactElement => {
         <Col start="sm" xs={12}>
           <Breadcrumb>
             <BreadcrumbElement iconType="transactionsInactive" text="TRANSACTIONS" />
+            {isDeeplinkedTx && <BreadcrumbElement text="DETAILS" color="placeHolder" />}
           </Breadcrumb>
         </Col>
       </Menu>
-      <Tab onChange={onTabChange} items={TRANSACTION_TABS} selectedTab={selectedTab} />
+      {!isDeeplinkedTx && <Tab onChange={onTabChange} items={TRANSACTION_TABS} selectedTab={path} />}
       <ContentWrapper>
         <Switch>
           <Route exact path={SAFE_ROUTES.TRANSACTIONS_QUEUE} render={() => <QueueTransactions />} />
           <Route exact path={SAFE_ROUTES.TRANSACTIONS_HISTORY} render={() => <HistoryTransactions />} />
+          {isDeeplinkedTx ? (
```

Comment:
Instead of rendering routes conditionally, maybe TRANSACTIONS and TRANSACTIONS_DETAILS could be two different routes?